packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -302,7 +302,12 @@ function View(props: ViewProps): React.ReactElement {
 		<Stack.Item grow={5} styles={contentViewStyles}>
 			<div
 				id="devtools-view-content"
-				style={{ width: "100%", height: "100%", overflowY: "auto" }}
+				style={{
+					width: "100%",
+					height: window.innerHeight,
```

Comment:
Doesn't this change the height to a static value (the specific inner height when this runs)? Not even sure if 100% is acting correctly with window resizes, but I'd expect this to definitely not work in that case, until the component is re-rendered (which doesn't happen on window resize... I think?).

Reply:
That is correct. You would have to bind an event handler on window resize events in order for this to work correctly. I would recommend not using window height.

Reply:
Was the intention behind this change to make the background scale to the height of the container? If so, I tackled that with this PR: #15539

Reply:
I need to test it out with the scenarios mentioned above (will update here once I do). 

Re: PR #15539, the divider between menu pane on left and on right, the divider still doesn't run to end of page based on the screenshot. Is that expected? Setting the height here did extend that divider to go all the way through. Maybe we need a combination of the two PRs?

Reply:
We may need to set height to `100%` here too to make the divider scale appropriately. But I would use 100% instead of the window height. Since the parent (with my changes) is now set to scale, I think it will work correctly.